scripts/exportexcel.py

```
@@ -0,0 +1,97 @@
+
+assets_dir = current_dir / "assets"
+outputs = [p for p in assets_dir.iterdir() if p.is_dir() if os.path.basename(str(p)) != "__pycache__"]
+
+sheets = JsonItem({})
+
+for output_dir in outputs:
+    tool = os.path.basename(output_dir.resolve())
+    filename = (output_dir / "output.json").resolve()
+    if not os.path.exists(str(filename)):
+        continue
+    with open(filename, 'r') as file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)